packages/grid/_modules_/grid/hooks/features/focus/useGridFocus.ts

```diff
@@ -28,6 +32,7 @@ export const useGridFocus = (apiRef: GridApiRef): void => {
           focus: { cell: { id, field }, columnHeader: null },
         };
       });
+      // TODO replace with constant
       apiRef.current.publishEvent('cellFocusChange');
```

Comment:
Don't we have that event already defined?

Reply:
There's only a `GRID_CELL_FOCUS` but it's mapped to the `focus` event.